server/src/main/scala/za/co/absa/atum/server/Constants.scala

```diff
@@ -22,10 +22,12 @@ object Constants {
 
     final val Api = "api"
     final val V1 = "v1"
+    final val V2 = "v2"
 
     final val CreatePartitioning = "createPartitioning"
     final val CreateOrUpdateAdditionalData = "createOrUpdateAdditionalData"
     final val CreateCheckpoint = "createCheckpoint"
+    final val GetFlowCheckpoints = "getFlowCheckpoints"
```

Comment:
I think these v2 endpoint paths should follow this naming convention 'get-flow-checkpoints'. Of course it's a topic for a team discussion.

Reply:
Okay. Once your envelope PR and FOrgiveness' PR will be merged to master, thus propagated here, I'll change it